### PR TITLE
Feature: header links in dropdown [NP-1235]

### PR DESCRIPTION
**Pull request by davidsauntson**
Displays the header links that appear above the search input in the 'More' dropdown on mobile.

 - update to greedy nav script to clone header links and put them in the dropdown
 - rationalise `navigation.scss` to follow BEM
 - rationalise `navigation-greedy-nav.scss` to follow BEM
 - update VRTs
 - handle no js by allowing nav elements to stack

Note that this work does not touch on any of the issues in the navigation design review ticket: https://citizensadvice.atlassian.net/browse/NP-1026

### CODE REVIEW

File: scss/6-components/_navigation_greedy-nav.scss [hunk 1]
```diff
@@ -11,43 +11,97 @@
...
+    &:active {
+      background-color: $cads-palette__white;
+      color: $cads-language__brand-primary;
+      border: 2px solid $cads-palette__heritage-yellow;
+      padding: calc(1rem - 2px);
+    }
+
+    &::after {
+      /* Copied from the icon font setup */
+      font-family: 'cads';
```
Comment (baseonmars) [inline scss/6-components/_navigation_greedy-nav.scss:74]: I made a placeholder called `%cads-icon` that could be used here.

File: scss/6-components/_navigation_greedy-nav.scss [hunk 2]
```diff
@@ -11,43 +11,97 @@
...
+  }
+
+  &__dropdown-toggle {
+    visibility: hidden;
+    position: absolute;
+    @include cads-button-defaults();
+
+    height: 100%;
+    font-weight: bold;
+    font-size: 1rem;
```
Comment (baseonmars) [inline scss/6-components/_navigation_greedy-nav.scss:53]: `$cads-spacing-4`?

File: scss/6-components/_navigation_greedy-nav.scss [hunk 3]
```diff
@@ -11,43 +11,97 @@
...
+      text-transform: none;
+      line-height: 1;
+
+      /* Better Font Rendering =========== */
+      -webkit-font-smoothing: antialiased;
+      -moz-osx-font-smoothing: grayscale;
+      content: '\0044';
+      display: inline-block;
+      transform: rotate(270deg);
+      font-size: 0.75rem;
```
Comment (baseonmars) [inline scss/6-components/_navigation_greedy-nav.scss:88]: `$cads-spacing-3`?

File: scss/6-components/_navigation_greedy-nav.scss [hunk 4]
```diff
@@ -11,43 +11,97 @@
...
-      &__dropdown {
-        &-toggle {
-          position: relative;
-        }
+    &:focus,
+    &:active {
+      background-color: $cads-palette__white;
+      color: $cads-language__brand-primary;
+      border: 2px solid $cads-palette__heritage-yellow;
+      padding: calc(1rem - 2px);
```
Comment (baseonmars) [inline scss/6-components/_navigation_greedy-nav.scss:69]: `$cads-spacing-4`

**Review by baseonmars — APPROVED** [review]
Few opportunities to use variables and a placeholder but overall nice. works well.

**Comment by davidsauntson** [comment]
> Few opportunities to use variables and a placeholder but overall nice. works well.

Thanks - yeah I was planning to tidy them up in the bug ticket that addresses all the incorrect spacing and styles from the previous development work, but I can do it now ❤️

**Comment by davidsauntson** [comment]
merging in now to get the alpha out - will address vars in NP-1026

**Pull request merged**